Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -147,45 +151,88 @@ class TouchIDSetting: Setting {
         // we can disable interaction of the switch and still handle tap events.
         let control = UISwitch()
         control.onTintColor = UIConstants.ControlTintColor
-        control.on = authInfo?.useTouchID ?? false
+        self.authInfo = KeychainWrapper.authenticationInfo()
```

Comment:
How come you moved this assignment out of the init call and into the `onConfigureCell` method? If we want to get information out of the keychain when we recycle the cells, why bother caching it in the setting object if we're invalidating it everytime we recycle?